step-templates/azure-database-export.json

```diff
@@ -0,0 +1,123 @@
+    {
+      "Id": "d5885f12-d921-4ce8-afc2-384f0b82daae",
+      "Name": "azDbExport.rgName",
+      "Label": "Resource Group Name",
+      "HelpText": null,
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "03b281f5-e1e2-42a8-bb43-317bc4981393",
+      "Name": "azDbExport.sqlSvrName",
+      "Label": "SQL Server Name",
+      "HelpText": "The name of the SQL Server to connect to.",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "e67ef4ab-3084-45e7-81df-7cda76baf2fc",
+      "Name": "azDbExport.dbName",
+      "Label": "Database Name",
+      "HelpText": "The name of the database you wish to export",
+      "DefaultValue": "",
+      "DisplaySettings": {}
+    },
+    {
+      "Id": "6da5679d-1c84-4fb4-8f91-22fd91b50baa",
+      "Name": "azDbExport.adminName",
+      "Label": "SQL Server Admin Name",
+      "HelpText": "The admin name of the SQL server containing the database you wish to export",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "44e1a2a3-6b9b-44a5-893d-d26f14166b52",
+      "Name": "azDbExport.adminPwd",
+      "Label": "Admin Password",
+      "HelpText": "The admin password of the SQL server containing the database you wish to export",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Sensitive"
+      }
+    },
+    {
+      "Id": "40715e88-4892-48c6-83f2-603c0193439b",
+      "Name": "azDbExport.blobAccessKey",
+      "Label": "Azure Blob Access Key",
+      "HelpText": "the access key (Shared Access Key or Storage Access Key) to grant access to the storage account",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Sensitive"
+      }
+    },
+    {
+      "Id": "fcb94050-5f81-410e-9ed9-c5f52653f9c6",
+      "Name": "azDbExport.accessKeyType",
+      "Label": "Access Key Type",
+      "HelpText": "The type of key the access key represents",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Select",
+        "Octopus.SelectOptions": "SharedAccessKey|Shared Access Key\nStorageAccessKey|Storage Access Key"
+      }
+    },
+    {
+      "Id": "a8631db7-556e-4532-aacd-595266c4347f",
+      "Name": "azDbExport.ContainerUri",
+      "Label": "Container Uri",
+      "HelpText": "The URI of the container to save the exported database in.  Format is: `https://{StorageAccountName}.blob.core.windows.net/{ContainerName}`",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "85686923-5e57-4b1d-8dd7-6b5ca263ba42",
+      "Name": "azDbExport.backupName",
+      "Label": "Backup Name",
+      "HelpText": "The name of the file being saved.  Defaults to DB Name",
```

Comment:
Can we add to the help text to explain that .bacpac will be appended to the filename.

Reply:
corrected in a639433